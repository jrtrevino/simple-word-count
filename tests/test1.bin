0000000: 01100001 01110011 01100100 00001010 01100001 01110011  asd.as
0000006: 01100100 00001010 01100001 01110011 01100100 00001010  d.asd.
