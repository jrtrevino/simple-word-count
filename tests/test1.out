        16        497 requirements
